CHANGES.rst

```diff
@@ -22,12 +27,19 @@ extract_1d
 - Remove pixel-by-pixel calls to wcs; copy input keywords to output for
   more types of input data. [#4685]
 
+- Added code to enable processing of NIRSPEC FIXEDSLIT AUTOWAVE data [#4626]
```

Comment:
Was this entry meant to go under `extract_2d`?

Reply:
Yes, thanks for spotting that